tests/avro/test_avro_producer.py

```diff
@@ -101,3 +101,26 @@ def test_produce_with_custom_registry_and_registry_url(self):
         schema_registry = MockSchemaRegistryClient()
         with self.assertRaises(ValueError):
             AvroProducer({'schema.registry.url': 'http://127.0.0.1:9001'}, schema_registry=schema_registry)
+
+    def test_produce_with_empty_value_identity(self):
+        schema_registry = MockSchemaRegistryClient()
+        producer = AvroProducer({}, schema_registry=schema_registry)
+        with self.assertRaises(ValueSerializerError):
+            producer.produce(topic='test', value='', key='not empty')
+
+    def test_produce_with_empty_key_identify(self):
```

Comment:
identify -> identity.

Maybe these should be renamed to be more clear why they're supposed to fail:
test_produce_with_empty_value_no_schema
test_produce_with_empty_key_no_schema
test_produce_with_empty_key_value_with_schema